From 2baaf66f9dec1b3c6c6025b4b44a37bba91f3f07 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH] formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 calculator.cpp | 4 +++-
 calculator.h   | 6 +++---
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index d93e35b..94c70cc 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,10 +2,12 @@
 
 int Calculator::Add (double a, double b)
 {
+
 	return a + b + 0.5;
+>>>>>>> cbcb06d (formatting: use tabs instead of spaces)
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.37.3.windows.1

